firmware/boron-ins-fsm/test/consoleFunctionTests.cpp

```diff
@@ -151,6 +150,7 @@ SCENARIO( "Change_IM21_Door_ID", "[change door id]" ) {
 SCENARIO( "Set Initial Timer", "[initial timer]" ) {
     GIVEN( "A starting initial timer of 10 milliseconds" ) {
         state1_max_time = 10000;
+        initial_timer_set("10");
```

Comment:
Can you please explain to me what is happening here?

I'm not sure I understand the relationship between `state1_max_time` and `initial_timer_set`. Do both of these need to be here?

Reply:
This is true, after the original EEPROM changes I made, only the initial_timer_set needed to be included. Since I am no longer using the mock EEPROM to store and get actual values, I removed the initial_timer_set for this test, as well as all of the following tests.